modules/Bio/EnsEMBL/Compara/Utils/DistanceMatrix.pm

```diff
@@ -96,13 +96,20 @@ sub members {
 sub distance {
 	my ($matrix, $s1, $s2, $new_distance) = @_;
 
-	return $matrix->{$s1}->{$s2} unless defined $new_distance;
+	if ( defined $new_distance ){
 	$matrix->{$s1}->{$s2} = $new_distance;
 	$matrix->{$s2}->{$s1} = $new_distance;
 	$matrix->{$s1}->{$s1} = 0;
 	$matrix->{$s2}->{$s2} = 0; # deal with diagonals
 	return $matrix;
 }
```

Comment:
It's sometimes hard to determine indentation on github - but you are allowed to correctly indent this curly brace, if it is not already correctly indented - to fit the function change.

Reply:
Well I did indent it originally but since the only chnages to those lines were whitespace changes they got dropped.

Reply:
So it this indented as it should be to fit the `if ( ) {` block? (I think no, since it doesn't show up as a change, and it would be odd if they were already indented). If not, please do :slightly_smiling_face: